### PR TITLE
chore: Linting tags

### PR DESCRIPTION
**Pull request by bbernays**
- [ ] Relates OR Closes #0000
- [ ] Added integration tests as described [here](https://docs.cloudquery.io/docs/developers/sdk/testing)

Output from integration tests:
More information about running the tests [here](../docs/contributing/e2e_tests.md)

```
$ go test -tags=integration ./...

```

### CODE REVIEW

File: resources/services/accessanalyzer/analyzers_mock_test.go [hunk 1]
```diff
@@ -1,3 +1,4 @@
+//go:build mock
 // +build mock
```
Comment (amanenk) [inline resources/services/accessanalyzer/analyzers_mock_test.go:2]: Should we remove old tags?

**Review by amanenk — APPROVED** [review]
LGTM
Should we use both types of tags or leave only latest variant?

File: resources/services/accessanalyzer/analyzers_mock_test.go (hunk 1 above)
Comment (bbernays) [reply resources/services/accessanalyzer/analyzers_mock_test.go:2]: I ran this gofmt command from the root of the repo and this is what it did  `gofmt -s -w .` ... gofmt behavior is usually taken as canonical

Comment (bbernays) [reply resources/services/accessanalyzer/analyzers_mock_test.go:2]: I have opened an issue to add gofmt to our ci https://github.com/cloudquery/.github/issues/1

**Review by disq — APPROVED** [review]

**Pull request merged**